docs/privacy-policy.rst

```diff
@@ -0,0 +1,304 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Most of Read the Docs is public-facing including
+user names, project names, and Documentation Sites.
+If your content is public-facing, third parties may access it.
+We do not sell that content; it is yours.
+
+
+Our use of cookies and tracking
+-------------------------------
+
+Cookies
+~~~~~~~
+
+Read the Docs uses cookies to make interactions with our service easy and meaningful.
+We use cookies to keep you logged in, remember your preferences,
+and provide information for future development of Read the Docs.
+
+A cookie is a small piece of text that our web server stores on your computer or mobile device,
+which your browser sends to us when you return to our site.
+Cookies do not necessarily identify you if you are merely visiting Read the Docs;
+however, a cookie may store a unique identifier for each logged in user.
+The cookies Read the Docs sets are essential for the operation of the website,
+or are used for performance or functionality.
+By using our website, you agree that we can place these types of cookies on your computer or device.
+If you disable your browser or device's ability to accept cookies,
+you will not be able to log in to Read the Docs.
+
+Google Analytics
+~~~~~~~~~~~~~~~~
+
+We use Google Analytics as a third party tracking service,
+but we don't use it to track you individually or collect your User Personal Information.
+We use Google Analytics to collect information about how our website performs
+and how our users, in general, navigate through and use Read the Docs.
+This helps us evaluate our users' use of Read the Docs;
+compile statistical reports on activity; and improve our content and website performance.
+
+Google Analytics gathers certain simple, non-personally identifying information over time,
+such as your IP address, browser type, internet service provider, referring and exit pages,
+time stamp, and similar data about your use of Read the Docs.
+We do not link this information to any of your personal information such as your user name.
+
+Read the Docs will not, nor will we allow any third party to,
+use the Google Analytics tool to track our users individually;
+collect any User Personal Information other than IP address;
+or correlate your IP address with your identity.
+Google provides further information about its own privacy practices and offers a
+`browser add-on to opt out of Google Analytics tracking <https://tools.google.com/dlpage/gaoptout>`_.
+
+
+How Read the Docs secures your information
+------------------------------------------
+
+Read the Docs takes all measures reasonably necessary to protect
+User Personal Information from unauthorized access, alteration, or destruction;
+maintain data accuracy; and help ensure the appropriate use of User Personal Information.
+We follow generally accepted industry standards to protect the personal information
+submitted to us, both during transmission and once we receive it.
```

Comment:
Should we enumerate this? SSL & password hashing I assume are the big ones?

Reply:
I don't think it is necessary especially since we don't (yet) require SSL on docs sites.